### DISCUSSION TITLE
NFT? It's not showing?

### DISCUSSION THREAD

**Discussion by ShubhSensei** [Cyfrin/foundry-full-course-cu#131] [Q&A] [answered]
why there is no NFT at all if I click on the NFT detail or is there any other way to look at it? 

<img width="889" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/61aacdb7-5c90-4b11-b243-ebf9df778b95">

**Comment by alymurtazamemon** [comment]
@SensationalShubham Import it in metamask by giving the contract address and token id.

**Reply by cromewar** [reply]
This is the way @SensationalShubham, does this make sense for you?

**Reply by ShubhSensei** [reply]
Yes, thanku @alymurtazamemon & @cromewar I got the NFTs. But now how can I import them on my real account on opensea.io 
when I open my account they are not there probably because I think I imported them on my Sepolia wallet test network. & I saw on Twitter that people are sharing the link of opensea.io instead of testnet.opensea.io

<img width="211" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/57eab99d-f9a5-405d-bfd1-030f123d620f">

**Reply by alymurtazamemon** [reply]
@SensationalShubham You need to mint on Arbutrum network in order to see them on opensea. Sepolia is a test network and there is opensea testnet but these NFTs do not show there.

**Reply by ShubhSensei** [reply]
Can you show me the exact path on how to mint/import them on Arbitrum plz? 

Also, as I already imported them on Sepolia testnet, can I still mint them on Arbitrum network?

**Reply by alymurtazamemon** [reply, marked as answer]
@SensationalShubham You must have seen 2 QR codes for each lesson, 1 is for sepolia and 1 is for the Arbitrum network. Both contain the same challenge for each lesson. Then why he created 2 QR codes? because the users can do all of the testing on the sepolia network and when they find the correct answer for each challenge then they can mint the same NFT on Arbitrum because it cost real Arbitrum ETH.

Just follow the same path you followed for the sepolia QR code and if you have real ETH on the Arbitrum network then you can mint it with the same answer.
